core/executor/Cargo.toml

```diff
@@ -14,7 +14,7 @@ serializer = { package = "substrate-serializer", path = "../serializer" }
 state_machine = { package = "substrate-state-machine", path = "../state-machine"  }
 runtime_version = { package = "sr-version", path = "../sr-version" }
 panic-handler = { package = "substrate-panic-handler", path = "../panic-handler" }
-wasmi = { version = "0.4.3" }
+wasmi = { path = "../../../wasmi", features = [ "threadsafe" ] }
```

Comment:
Instead of the local path, could you put the git ref? This would make it buildable for other contributors.